content/en/docs/contributing/_index.md

```diff
@@ -35,14 +36,12 @@ To get invites you can subscribe to [our mailing list](https://groups.google.com
 
 We have 2 regular repeating meetings:
 
-* our daily stand-up meetings [on Google Meet](
-https://meet.google.com/eum-fyvt-xpa) at 10:30 London time every weekday 
-* our bi-weekly developer meetings [are on Google Meet](https://meet.google.com/abp-bwhk-wxc) 17:00 London time every 2 weeks (check calendar invites or [meeting notes](https://docs.google.com/document/d/1Tc5t6ylY9dhXAan1OjOoldeaoys1Yh4Ir710ATfBa5U)). 
-
-You can check what time this is where you are using [`everytimezone.com`](https://everytimezone.com/).
+* daily stand-up meetings [on Google Meet](https://meet.google.com/eum-fyvt-xpa) at [10:30 London time](http://www.thetimezoneconverter.com/?t=10:30&tz=Europe/London) every weekday
+* bi-weekly developer meetings [on Google Meet](https://meet.google.com/abp-bwhk-wxc) at [17:00 London time](http://www.thetimezoneconverter.com/?t=17:00&tz=Europe/London) (for dates, check calendar invites or [meeting notes](https://docs.google.com/document/d/1Tc5t6ylY9dhXAan1OjOoldeaoys1Yh4Ir710ATfBa5U))
```

Comment:
I've also added clickable direct timezone conversions, inspired by what the Kubernetes project does